src/diffusers/pipelines/pipeline_utils.py

```diff
@@ -1013,7 +1014,7 @@ def load_module(name, value):
         from diffusers import pipelines
 
         # 6. Load each module in the pipeline
-        for name, (library_name, class_name) in init_dict.items():
+        for name, (library_name, class_name) in tqdm(init_dict.items(), desc="Loading pipeline components..."):
```

Comment:
Help me understand this. 

When we do `DiffusionPipeline.from_pretrained(ckpt_id)` it already does the series of progress bars for the individual components, no? Then why is this being added?

Reply:
@sayakpaul I guess to see some progress while the model is loading, even if it's cached.